mongodbatlas/access_tracking_test.go

```diff
@@ -0,0 +1,186 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mongodbatlas
+
+import (
+	"fmt"
+	"net/http"
+	"testing"
+
+	"github.com/go-test/deep"
+)
+
+func TestAccessTracking_ListByCluster(t *testing.T) {
+	client, mux, teardown := setup()
+	defer teardown()
+
+	mux.HandleFunc(fmt.Sprintf("/api/atlas/v1.0/groups/%s/dbAccessHistory/clusters/%s", groupID, clusterName), func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, http.MethodGet)
+		fmt.Fprint(w, `{
+			  "accessLogs": [
+					{
+					  "authResult": true,
+					  "authSource": "admin",
+					  "groupId": "1",
+					  "hostname": "cluster0-shard-00-00-c01ab.mongodb.net:27017",
+					  "clusterName": "globalCluster",
+					  "ipAddress": "123.45.0.1",
+					  "logLine": "2019-07-25T19:14:42.484+0000 I  ACCESS   [conn2167] Successfully authenticated as principal jon-snow on admin from client 123.45.0.1:8080",
+					  "timestamp": "Sun Jul 25 9:14:42 EDT 2019",
+					  "username": "jon-snow"
+					},
+					{
+					  "authResult": true,
+					  "authSource": "admin",
+					  "failureReason": "UserNotFound: Could not find user \"jane-doe\" for db \"admin\"",
+					  "groupId": "1",
+					  "hostname": "cluster0-shard-00-00-c01ab.mongodb.net:27017",
+					  "clusterName": "globalCluster",
+					  "ipAddress": "123.45.2.2",
+					  "logLine": "2019-07-25T19:13:39.316+0000 I  ACCESS   [conn1893] SASL SCRAM-SHA-1 authentication failed for jane-doe on admin from client 123.45.2.2:51842 ; UserNotFound: Could not find user \"jane-doe\" for db \"admin\"",
+					  "timestamp": "Sun Jul 25 9:14:42 EDT 2019",
+					  "username": "jane-doe"
+					}]
+		}`)
+	})
+
+	opts := &AccessLogOptions{
+		Start:      "Sun Jul 20 9:14:42 EDT 2019",
```

Comment:
[nit] the docs say this is epoch, can you please either fix this or fix the docs?